tsfc/fiatinterface.py

```diff
@@ -145,21 +227,8 @@ def _(element, vector_is_mixed):
         # Can't use create_element here because we're going to modify
         # it, so if we pull it from the cache, that's bad.
```

Comment:
Instead you can use `element = create_element(element, vector_is_mixed)` I think